src/apic-extension/azext_apic_extension/tests/latest/test_integration_command.py

```diff
@@ -0,0 +1,45 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+from .utils import ApicServicePreparer, ApimServicePreparer
+from .constants import TEST_REGION, USERASSIGNED_IDENTITY
+
+# if USERASSIGNED_IDENTITY is set, enable_system_assigned_identity is False, otherwise use system assigned identity
+enable_system_assigned_identity = False if USERASSIGNED_IDENTITY else True
+
+class IntegrationCommandTests(ScenarioTest):
+    # TODO: change the location to TEST_REGION when the APIC resource provider is available in all regions
+    @ResourceGroupPreparer(name_prefix="clirg", location="centraluseuap", random_name_length=32)
+    @ApicServicePreparer(enable_system_assigned_identity=enable_system_assigned_identity)
+    @ApimServicePreparer(enable_system_assigned_identity=enable_system_assigned_identity)
+    def test_integration_create_apim(self):
+        # prepare test data
+        self.kwargs.update({
+          'integration_name': self.create_random_name(prefix='cli', length=8)
+        })
+
+        if enable_system_assigned_identity:
+            self.cmd('az apic integration create apim -g {rg} -n {s} --azure-apim {apim_name} -i {integration_name}')
+        else:
+            # TODO: ugly codes, need to find a better way to handle this
+            import re
+            self.kwargs.update({
+                # remove curly braces from usi_id
+                'usi_id': re.sub(r'^\{|\}$', '', self.kwargs['usi_id'])
```

Comment:
minor: Can you give me an example of `self.kwargs['usi_id']`? I don't quite understand why there's `{}` in it.

Reply:
[line 429 in utils.py](https://github.com/blackchoey/azure-cli-extensions/pull/86/files/31dcf4a86b949379be1b3878b792dcfc4dbe4d50#diff-339b0f43ee39d64a942c2709c421e4c548e99cac109fb962c00d1bdfd68256af:~:text=self.test_class_instance.cmd(%27az%20apic%20update%20%2D%2Dname%20%7B%7D%20%2Dg%20%7B%7D%20%2D%2Didentity%20%7B%7Btype%3AUserAssigned%2Cuser%2Dassigned%2Didentities%3A%7B%7D%7D%7D%27.format(self.test_class_instance.kwargs%5B%27s%27%5D%2C%20group%2C%20self.test_class_instance.kwargs%5B%27usi_id%27%5D))) needs a usi_id with `{}`, so I configured the value in github secrets with `{}`.

Reply:
We may just set the value  without `{}`, and add `{}` in the utils.py? It seems more nature than removing `{}` here.